### DISCUSSION TITLE
请问如何在windows环境使用？

### DISCUSSION THREAD

**Discussion by 1004217128** [xjasonlyu/tun2socks#217] [Q&A]
如何在windows环境使用？
我执行了>tun2socks -device Trojan -proxy socks5://127.0.0.1:10808
反馈：
2023/01/22 23:06:18 Failed to create private namespace: 拒绝访问。 (Code 0x00000005)
2023/01/22 23:06:18 Failed to take device installation mutex: 拒绝访问。 (Code 0x00000005)
time="2023-01-22T23:06:18+08:00" level=fatal msg="[ENGINE] failed to start: create tun: Error creating interface: Access is denied."

**Comment by yundmb** [comment]
你这很明显需要管理员权限

**Reply by Linweisong666** [reply]
问一下你后面遇到无限回环的问题怎么解决的
